spec/controllers/users_controller_spec.rb

```diff
@@ -1232,6 +1232,7 @@ def post_user
           guardian = Guardian.new(user)
           guardian.stubs(:ensure_can_edit!).with(user).raises(Discourse::InvalidAccess.new)
           Guardian.stubs(new: guardian).with(user)
+          Guardian.stubs(new: guardian).with(nil)
```

Comment:
Okay, I updated the spec via https://github.com/discourse/discourse/commit/6df5b38b54d265fef0e6a9749ff89839db898374